recipes/kmsxx/meta.yaml

```
@@ -0,0 +1,93 @@
+{% set version = "2025.08.0" %}
```

```suggestion
{% set version = "0.0.0.2025.08.0" %}
```
After some discussion with other maintainers, it has been suggested to format calver like this in case the project ever decides to release proper tags.